README.md

```diff
@@ -1,44 +1,38 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+# React Date Range Picker
+react-daterange-picker can be used to pick a date range in react and specify a data refresh interval.
+## Demo
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-body-1.PNG)
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-body-2.PNG)
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-timer-1.PNG)
+## Dependencies
+ - install the [package.json](https://github.com/iamPres/react-daterange-picker/blob/master/package.json) dependencies with npm
+
+## Example Usage
+```import React from "react";
+import ReactDOM from "react-dom";
+import { Layout } from "./Components/Layout.tsx";
```

Comment:
You'll want the installation to be from a single top-level file, something like:
```
import DateRangePicker from "react-date-range-picker";
```

You can do that by having a `src/index.ts` file that strictly imports and exports - likely something like importing the layout object and exporting it as DateRangePicker.

Reply:
src/index.ts wouldnt be top level would it?

Reply:
i think i figured it out